pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1509,7 +1526,11 @@ def add_layer(
         name : str
             Layer name.
         layer_type : str, optional
-            Layer type. Options are ``"S"``, ``"D"``, and ``"G"``. The default is ``"S"``.
+            Layer type. Options are
```

Comment:
```suggestion
            Layer type. Options are:
            
```
Use a semicolon to introduce the list of options. You also must have a blank line before the first item in the list for it to render correctly in the documentation.